versions/1.1.0-dev.md

```diff
@@ -282,6 +282,28 @@ The extensions may or may not be supported by the available tooling, but those m
 Overlay files MAY choose to follow the convention of a `purpose.overlay.yaml` file naming pattern.
 Other file naming conventions are also supported.
 
+### RFC9535 compliance
+
+[[RFC9535|RFC9535]] is a recent specification and libraries implementing JSONPath support might predate the RFC. Those libraries might differ entirely (expressions syntax is incompatible), or implement additional capabilities (superset of the RFC). A tool or library MUST fully implement [[RFC9535|RFC9535]] when parsing and expanding JSONPath query expressions to be compliant with the Overlay specification.
+
+Users SHALL refrain from using additional capabilities offered by the tool's JSONPath implementation to avoid any interoperability issues with their Overlay Documents.
+
+In case the tool being use is not fully compliant with RFC9535, users MIGHT have to update some JSONPath query expressions to accommodate for the implementation in use.
+
+### Example of updates to JSONPath
+
+This example JSONPath query expression:
+
+```jsonpath
+$.paths.*.get.parameters[?@.name=='filter' && @.in=='query']
+```
+
+Might require additional optional parenthesis with some implementations like so:
+
+```jsonpath
+$.paths.*.get.parameters[?(@.name=='filter' && @.in=='query)']
+```
+
```

Comment:
This too seems inappropriate for the spec, though I agree it is useful information. Could we put this in the README instead, as a subsection of "Tools that Support Overlays"?

Reply:
This example subsection should move to the README.

The compliance section should be in the spec.

Reply:
moved this paragraph and the example to the readme